components/DarkModeToggler/DarkModeToggler.tsx

```diff
@@ -0,0 +1,31 @@
+import { Switch } from '@headlessui/react'
+import {useTheme} from 'next-themes';
```

Comment:
```suggestion
import { useTheme } from 'next-themes';
```
Why is the formatter not catching this, btw?